fastlane/Fastfile

```diff
@@ -73,29 +73,36 @@ platform :android do
   desc "Upload and close a release"
   lane :deploy do |options|
     version = android_get_version_name(gradle_file: gradle_file_path)
-    is_snapshot_version = version.end_with?("-SNAPSHOT")
-    puts "Deploying #{version}"
-    gradle(
-      tasks: [
-          "androidSourcesJar", "androidJavadocJar", "publish", "closeAndReleaseRepository"
-      ],
-      properties: {
-        "signing.keyId" => ENV['GPG_SIGNING_KEY_ID_NEW'],
-        "signing.password" => ENV['GPG_SIGNING_KEY_PW_NEW'],
-        "signing.secretKeyRingFile" => "/home/circleci/.gnupg/secring.gpg",
-        "mavenCentralUsername" => ENV['SONATYPE_NEXUS_TOKEN_USERNAME'],
-        "mavenCentralPassword" => ENV['SONATYPE_NEXUS_TOKEN_PASSWORD'],
-        "RELEASE_SIGNING_ENABLED" => true,
-      }
-    )
-    if !is_snapshot_version then github_release(version: version) end
+  	is_snapshot_version = version.end_with?("-SNAPSHOT")
+  	gradleProperties = { "signing.keyId" => ENV['GPG_SIGNING_KEY_ID_NEW'],
+  		"signing.password" => ENV['GPG_SIGNING_KEY_PW_NEW'],
+  		"signing.secretKeyRingFile" => "/home/circleci/.gnupg/secring.gpg",
+  		"mavenCentralUsername" => ENV['SONATYPE_NEXUS_TOKEN_USERNAME'],
+  		"mavenCentralPassword" => ENV['SONATYPE_NEXUS_TOKEN_PASSWORD'],
+  		"RELEASE_SIGNING_ENABLED" => true
+  	}
+  	puts "Deploying #{version}"
+  	gradle(
```

Comment:
@vegaro not sure why the formatting looks so awful in github, looks normal in androidstudio

Reply:
![Screen Shot 2021-05-12 at 7 15 03 PM](https://user-images.githubusercontent.com/2272924/118067972-68b23180-b356-11eb-8cfb-a7b626a6bfc7.png)

Reply:
Yeah I think there's a mix between tabs and spaces https://raw.githubusercontent.com/RevenueCat/purchases-android/fix-fastlane/fastlane/Fastfile

<img width="150" alt="Screen Shot 2021-05-13 at 1 26 41 PM" src="https://user-images.githubusercontent.com/664544/118183363-ddd14580-b3ee-11eb-972a-d1f76b08cfa9.png">